digital-land/config-manager
# Internal error graph

## What type of PR is this? (check all applicable)

-  Feature

## Description

Adds a timeseries for internal errors (currently only unknown errors)

![image](https://github.com/user-attachments/assets/2b3c85ae-59e4-49ea-8504-dc9dc1ed7e35)

Currently, `operational_issues` has over 345000 rows. Due to how datasette returns data, we need get the data with an offset 1000 so it does a lot of requests and takes up to 3 minutes to perform. Ideally, this the limit needs to be changed or some optimisation done.

Linting made a lot of files changed, but the updated code for this features is in:

- blueprints/reports/ views.py
- data_access/ summary_queries.py
- reporting/overview.html

## Related Tickets & Documents


- Ticket Link: https://dluhcdigital.atlassian.net/jira/software/projects/DATA/boards/229?selectedIssue=DATA-22
- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

Go to https://config-manager-prototype.herokuapp.com/reporting/overview and see the graph at the bottom of the page

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

-  No, and this is why: no tests are currently done on config-manager

## [optional] Are there any post deployment tasks we need to perform?

## [optional] Are there any dependencies on other PRs or Work?
